diff --git a/config/config.exs b/config/config.exs
index f5dc26cb..3aeeb775 100644
--- a/config/config.exs
+++ b/config/config.exs
@@ -57,13 +57,6 @@ config :mobilizon, Mobilizon.Web.Endpoint,
   cache_static_manifest: "priv/static/manifest.json",
   has_reverse_proxy: true
 
-config :mime, :types, %{
-  "application/activity+json" => ["activity-json"],
-  "application/ld+json" => ["activity-json"],
-  "application/jrd+json" => ["jrd-json"],
-  "application/xrd+xml" => ["xrd-xml"]
-}
-
 # Upload configuration
 config :mobilizon, Mobilizon.Web.Upload,
   uploader: Mobilizon.Web.Upload.Uploader.Local,
